pkg/test/framework/components/istio/operator.go

```diff
@@ -215,6 +212,24 @@ func deploy(ctx resource.Context, env *kube.Environment, cfg Config) (Instance,
 			return nil, err
 		}
 	}
+	// If UseLegacyRemote is false, we are under central Istiod which uses remote KUBECONFIG
```

Comment:
Remote the use of "central istiod" here ... most people think of "config clusters" when they think of central istiod. Here we're just using the new style of remote clusters.

Reply:
thanks for the suggestion, fixing